CHANGELOG.asciidoc

```diff
@@ -29,11 +29,6 @@ https://github.com/elastic/beats/compare/v6.4.1...6.4[Check the HEAD diff]
 
 *Affecting all Beats*
 
-- Fix a race condition with the `add_host_metadata` and the event serialization. {pull}8223[8223] {pull}8653[8653]
-- Fix race condition when publishing monitoring data. {pull}8646[8646]
-- Fix bug in loading dashboards from zip file. {issue}8051[8051]
```

Comment:
What about this one?

Reply:
Thanks, I added it back. Not sure why it fell between the cracks.